src/main/java/spoon/support/compiler/jdt/JDTTreeBuilder.java

```diff
@@ -935,7 +936,10 @@ public boolean visit(BreakStatement breakStatement, BlockScope scope) {
 
 	@Override
 	public boolean visit(CastExpression castExpression, BlockScope scope) {
-		context.casts.add(this.references.buildTypeReference(castExpression.type, scope));
+		CastInfo ci = new CastInfo();
+		ci.nrOfBrackets = ((castExpression.bits >>> 21) & 0xF);
```

Comment:
ok this is a low level one ;) Could you explain the operation in a comment please?